src/content/styles/Footer.scss

```diff
@@ -17,7 +17,7 @@ Pulled from: https://bbbootstrap.com/snippets/social-media-icons-footer-54412302
 
 #sg-logo {
     width: 120px;
-    height: max-content;
+    height: auto;
```

Comment:
You're setting the width so it should automatically scale the height to the correct aspect ratio anyways? Is there a specific use case for setting the height to `auto`?

Reply:
I believe `auto` just allows it to automatically scale to the proper aspect ratio. it may be redundant in this case but to stay on the safe side since I'm defining the width I should define the height.